tests/apps/core/test_core_models_users.py

```diff
@@ -92,7 +100,22 @@ def test_models_users_username_ascii(self):
         self.assertEqual(
             context.exception.messages,
             [
-                "Username must contain only lower case letters, numbers, underscores and hyphens."
+                "Username must contain only lower case letters, numbers, hyphens, "
+                "periods and underscores."
+            ],
+        )
+
+    @override_settings(USERNAME_REGEX=r"^[a-z0-9_-]+$")
+    def test_models_users_username_override_regex(self):
+        """It should be possible to override the username regex to allow custom characters."""
```

Comment:
I think the doc string does not describe well the behavior tested as you are removing period from the regex rather to allow new chars.

Reply:
You're right! I improved the docstring